### PR TITLE
tags: module externalisation to new package

### PR DESCRIPTION
**Pull request by hachreak**
* INCOMPATIBILITY Removes 'tags' module that has been externalised to
  separate Python package called 'invenio_tags'.
  Migration can be done by running `pip install invenio_tags` and
  adding 'invenio_tags' to PACKAGES.

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: invenio/modules/tags/__init__.py [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-#
-# Invenio is distributed in the hope that it will be useful, but
-# WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-# General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with Invenio; if not, write to the Free Software Foundation, Inc.,
-# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
-"""
```
Comment (jirikuncar) [inline invenio/modules/tags/__init__.py]: please keep a warning message here

**Comment by jirikuncar** [comment]
- [ ] add `invenio.modules.tags` to `EXCLUDED_PACKAGES` in `invenio.base.config`

**Comment by hachreak** [comment]
ping @jirikuncar

File: .gitignore [hunk 2]
```diff
@@ -36,4 +36,3 @@ invenio/base/translations/*/LC_MESSAGES/messages.mo
 missing
 node_modules/
 org.eclipse.core.resources.prefs
-tags
```
Comment (jirikuncar) [inline .gitignore]: this should stay ;)

**Comment by hachreak** [comment]
@jirikuncar done

**Pull request merged**